local-container-endpoints/version/version.go

```diff
@@ -21,7 +21,7 @@ package version
 // you are doing.
 
 // Version is the version of ECS Local Container Endpoints
-const Version = "1.0.1"
+const Version = "1.4.0"
```

Comment:
Note: this file had not updated since the 1.0.1 release. Not sure if we still use this for anything major, but ran `go run gen/version-gen.go` from the `local-container-endpoints/version` dir to update this file.